pyproject.toml

```diff
@@ -12,10 +12,9 @@ description = "A project (and object) for storing, manipulating, and converting
 readme = "README.md"
 license = { text = "MIT" }
 authors = [ { name = "Open Force Field Initiative", email = "info@openforcefield.org" } ]
+requires-python = ">=3.11"
```

Comment:
Note to reviewer: this is effectively the only substantive change in this PR. Everything else is automated and should amount only to formatting changes